Zend/zend_compile.c

```diff
@@ -3770,12 +3842,12 @@ void zend_compile_dynamic_call(znode *result, znode *name_node, zend_ast *args_a
 }
 /* }}} */
 
-static inline bool zend_args_contain_unpack_or_named(zend_ast_list *args) /* {{{ */
+static inline bool zend_args_contain_unpack_or_named_or_partial(zend_ast_list *args) /* {{{ */
```

Comment:
This might want a better name, any suggestions ?

Reply:
`zend_args_contain_special` maybe? ^^